### PR TITLE
doc(release): instructions for update master branch version and etc

### PR DESCRIPTION
**Pull request by Bobgy**
/assign @jingzhang36

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -209,9 +209,24 @@ fill in the description. Detailed steps:
...
        ```
 
-       See the [Change Log](https://github.com/kubeflow/pipelines/blob/master/CHANGELOG.md)
+       See the [Change Log](https://github.com/kubeflow/pipelines/blob/$TAG_NAME/CHANGELOG.md)
        </pre>
 
+1. Update master branch to the same version.
+    ```bash
+    export TAG_NAME=<TAG_NAME>
+    git co master
```
Comment (jingzhang36) [inline RELEASE.md:218]: should we use the full command here, i.e., git checkout instead of git co. E.g., in my git version, co is not a valid command

Comment (Bobgy) [reply RELEASE.md]: good point! Updated

**Pull request merged**